<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TKCBKNG" failures="2" tests="5" name="Default test" time="66.16" errors="0" timestamp="2025-04-02T00:32:23 IST">
  <testcase classname="pomTests.BlogTC" name="openBlogPage" time="1.182"/>
  <testcase classname="pomTests.BlogTC" name="LatestArticleSection" time="9.076"/>
  <testcase classname="pomTests.BlogTC" name="MostViewedArticleSection" time="6.222"/>
  <testcase classname="pomTests.BlogTC" name="WriteaCommentSection" time="30.02">
    <failure type="com.microsoft.playwright.TimeoutError" message="Error {
  message=&amp;apos;Timeout 30000ms exceeded.
  name=&amp;apos;TimeoutError
  stack=&amp;apos;TimeoutError: Timeout 30000ms exceeded.
    at ProgressController.run (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\progress.js:87:26)
    at Frame.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\frames.js:995:23)
    at FrameDispatcher.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\frameDispatcher.js:152:30)
    at FrameDispatcher._handleCommand (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:84:40)
    at DispatcherConnection.dispatch (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:351:39)
}
Call log:
- waiting for locator(&amp;quot;#entry_210904&amp;quot;)
">
      <![CDATA[com.microsoft.playwright.TimeoutError: Error {
  message='Timeout 30000ms exceeded.
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
    at ProgressController.run (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\progress.js:87:26)
    at Frame.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\frames.js:995:23)
    at FrameDispatcher.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\frameDispatcher.js:152:30)
    at FrameDispatcher._handleCommand (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:84:40)
    at DispatcherConnection.dispatch (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:351:39)
}
Call log:
- waiting for locator("#entry_210904")

at com.microsoft.playwright.impl.WaitableResult.get(WaitableResult.java:52)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:120)
at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:129)
at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:106)
at com.microsoft.playwright.impl.FrameImpl.clickImpl(FrameImpl.java:252)
at com.microsoft.playwright.impl.FrameImpl.lambda$click$8(FrameImpl.java:243)
at com.microsoft.playwright.impl.LoggingSupport.lambda$withLogging$0(LoggingSupport.java:36)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:89)
at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:35)
at com.microsoft.playwright.impl.FrameImpl.click(FrameImpl.java:243)
at com.microsoft.playwright.impl.LocatorImpl.click(LocatorImpl.java:140)
at com.microsoft.playwright.Locator.click(Locator.java:2348)
at com.lt.pages.BlogPage.WriteComment(BlogPage.java:184)
at pomTests.BlogTC.WriteaCommentSection(BlogTC.java:78)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: com.microsoft.playwright.TimeoutError: Error {
  message='Timeout 30000ms exceeded.
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
    at ProgressController.run (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\progress.js:87:26)
    at Frame.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\frames.js:995:23)
    at FrameDispatcher.click (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\frameDispatcher.js:152:30)
    at FrameDispatcher._handleCommand (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:84:40)
    at DispatcherConnection.dispatch (C:\Users\dasun\AppData\Local\Temp\playwright-java-14162533202240962991\package\lib\server\dispatchers\dispatcher.js:351:39)
}
Call log:
- waiting for locator("#entry_210904")

at com.microsoft.playwright.impl.Connection.dispatch(Connection.java:252)
at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:209)
at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:118)
... 39 more
]]>
    </failure>
  </testcase> <!-- WriteaCommentSection -->
  <testcase classname="pomTests.BlogTC" name="ReadMoreButtonArticleSection" time="5.214">
    <failure type="java.lang.AssertionError" message="expected [eget nunc lobortis mattis aliquam faucibus purus in massa tempor] but found [Jolio Balia  16 comments  23829  25 Feb 2019]">
      <![CDATA[java.lang.AssertionError: expected [eget nunc lobortis mattis aliquam faucibus purus in massa tempor] but found [Jolio Balia  16 comments  23829  25 Feb 2019]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.lt.pages.BlogPage.BusinessArticle(BlogPage.java:174)
at pomTests.BlogTC.ReadMoreButtonArticleSection(BlogTC.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- ReadMoreButtonArticleSection -->
</testsuite> <!-- Default test -->
